boot/bootutil/src/loader.c

```diff
@@ -590,6 +590,7 @@ boot_validate_slot(int slot)
              * continue booting from slot 0.
              */
         }
+        BOOT_LOG_ERR("Image in slot%d is not valid!", slot);
```

Comment:
I'd put a space after "slot", but this seems fine to me

Reply:
Added the space!